Neos.Fusion/Resources/Private/Fusion/Root.fusion

```diff
@@ -1,291 +1 @@
-}
-
-# Renders an URI pointing to a controller/action
-#
-# Usage:
-# uri = Neos.Fusion:ActionUri {
-#   package = 'Some.Package'
-#   controller = 'Standard'
-#   action = 'index'
-# }
-#
-prototype(Neos.Fusion:ActionUri) {
-  @class = 'Neos\\Fusion\\FusionObjects\\ActionUriImplementation'
-  request = ${request}
-  additionalParams = Neos.Fusion:DataStructure
-  arguments = Neos.Fusion:DataStructure
-  argumentsToBeExcludedFromQueryString = Neos.Fusion:DataStructure
-
-  @exceptionHandler = 'Neos\\Fusion\\Core\\ExceptionHandlers\\AbsorbingHandler'
-}
-
-# Renders a link pointing to a controller/action
-#
-# Usage:
-# link = afx`
-#    <Neos.Fusion:Link.Action class="action-link" href.package="Some.Package" href.controller="Standard" href.action="index">
-#        Some Link
-#    </Neos.Fusion:Link.Action>
-# `
-#
-prototype(Neos.Fusion:Link.Action) < prototype(Neos.Fusion:Component) {
-    href = Neos.Fusion:ActionUri
-    content = ''
-
-    renderer = Neos.Fusion:Tag {
-        tagName = 'a'
-        attributes {
-            @ignoreProperties = ${['content']}
-            @apply.props = ${props}
-        }
-        content = ${props.content}
-    }
-}
-
-# Renders a link pointing to a resource
-#
-# Usage:
-# link = afx`
-#    <Neos.Fusion:Link.Resource class="resource-link" href.path="resource://Some.Package/Public/Images/SomeImage.png">
-#        Some Link
-#    </Neos.Fusion:Link.Resource>
-# `
-#
-prototype(Neos.Fusion:Link.Resource) < prototype(Neos.Fusion:Component) {
-    href = Neos.Fusion:ResourceUri {
-        localize = true
-    }
-    content = ''
-
-    renderer = Neos.Fusion:Tag {
-        tagName = 'a'
-        attributes {
-            @ignoreProperties = ${['content']}
-            @apply.props = ${props}
-        }
-        content = ${props.content}
-    }
-}
-
-# Render a content fragment without any additional markup
-# to allow placing conditions for multiple tags at once.
-#
-# Usage:
-# renderer = afx`
-#    <Neos.Fusion:Fragment @if.isEnabled={props.enable}>
-#       <h1>Example</h1>
-#       <h2>Content</h2>
-#    </Neos.Fusion:Fragment>
-# `
-prototype(Neos.Fusion:Fragment) < prototype(Neos.Fusion:Component) {
-  renderer = ${props.content}
-}
-
-# These are globally applied cache identifiers.
-# If you don't make @cache.entryIdentifiers another prototype (like a Neos.Fusion:DataStructure)
-# they will be rendered as this prototype, which means everything in here is added to ALL cached
-# Fusion parts.
-# The baseUri contains the protocol (http/https), domain and port.
-#
-# WARNING: Do not delete this prototype, it is used inside of \Neos\Fusion\Core\Cache\RuntimeContentCache::buildCacheIdentifierValues() as a fallback.
-#
-prototype(Neos.Fusion:GlobalCacheIdentifiers) < prototype(Neos.Fusion:DataStructure) {
-  format = ${request.format}
-  baseUri = ${String.toString(BaseUri.getConfiguredBaseUriOrFallbackToCurrentRequest(request.httpRequest))}
-}
+include: **/*.fusion
```

Comment:
As Marc mentioned, I think we should move all Fusion files from root to a Prototypes folder. Then we can include only the folders we want and maybe remove `Deprecated` from the include.

Reply:
Done